README.md

```diff
@@ -18,61 +20,78 @@ Citation:
 
 ## Quickstart Guide
 
-### Installation
+To immediately use a pre-trained Howl model for inference, we provide the `client` API.
+
+```
+TODO: Add example here
+```
 
-A proper Pip package is coming soon. 
+## Training Guide
+
+### Installation
 
 1. `git clone https://github.com/castorini/howl && cd howl`
 
 2. Install [PyTorch](https://pytorch.org) by following your platform-specific instructions.
 
 3. Install PyAudio and its dependencies through your distribution's package system.
 
-4. `pip install -r requirements.txt` (some apt packages might need to be installed)
+4. `pip install -r requirements.txt -r requirements_training.txt` (some apt packages might need to be installed)
```

Comment:
does this work `pip install -r requirements.txt requirements_training.txt`?

Reply:
Yeah you can specify multiple

Reply:
You have two `-r` so I was asking